plugins/catalog-graph/src/components/CatalogGraphPage/CatalogGraphPage.tsx

```diff
@@ -105,6 +107,8 @@ const useStyles = makeStyles(
 
 export const CatalogGraphPage = (props: {
   relationPairs?: RelationPairs;
+  renderNode?: Types.RenderNodeFunction<EntityNode>;
```

Comment:
Can we add these instead as 
```
graphProps: Partial<EntityRelationsGraphProps>
```
similar to the CatalogGraphCard work? You can then pass it as `{...props}` to the EntityRelationsGraph and it allow for more than just the two props.

As I mentioned in my comment on the issue, I think there's also some work needed to shift some initialState into that prop, but it shouldn't be part of this PR.

Reply:
@sennyeya  Updated. PTAL.

Reply:
@sennyeya Once we merge this PR, maybe we can create one more issue for moving initialState into prop.
Happy to contribute on same.